--- openh323proxy/opengate.cxx.orig	2002-11-28 11:44:54.000000000 +0100
+++ openh323proxy/opengate.cxx	2003-04-19 17:21:27.000000000 +0200
@@ -178,7 +178,7 @@
 	// Try and read settings from the configuration file, if they are not
 	// there using sensible default
 
-	PConfig Cfg("System");	
+	PConfig Cfg("/etc/opengate_proxy.init", "System");
 
 	// Set the local address to bind to, this should be useful in multi-homed environments
 	PString AkaLocalAddr = Cfg.GetString( "Local Address", "*" );
